### PR TITLE
Allow to switch to an invisible iframe (closes #1326)

### LINKED ISSUE #1326: Allow to switch to an invisible iframe
Now if you call 
```js
await t.switchToIframe('iframe-selector');
```
and an appropriated iframe is invisible TestCafe raises an error like `can't switch to an invisible iframe`.
We shouldn't raise an error in this case. For example it should be possible to evaluate a client function in an uninvisible iframe

### PR DESCRIPTION
**Pull request by AlexanderMoskovkin**

### CODE REVIEW

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

File: src/client/driver/driver-link/child.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
-import { CurrentIframeIsNotLoadedError, CurrentIframeNotFoundError, CurrentIframeIsInvisibleError } from '../../../errors/test-run';
+import {
+    CurrentIframeIsNotLoadedError,
+    CurrentIframeNotFoundError,
+    CurrentIframeIsInvisibleError
+} from '../../../errors/test-run';
 import sendMessageToDriver from './send-message-to-driver';
 import { ExecuteCommandMessage, ConfirmationMessage, TYPE as MESSAGE_TYPE } from './messages';
 import DriverStatus from '../status';
+import { doesCommandRequireVisibleElement } from '../../../test-run/commands/utils';
```
Comment (inikulin) [inline src/client/driver/driver-link/child.js]: `isElementVisibilityRequired`

File: src/errors/test-run/templates.js [hunk 2]
```diff
@@ -207,7 +207,7 @@ export default {
     `),
 
     [TYPE.currentIframeIsInvisibleError]: err => markup(err, `
-        The iframe in which the test is currently operating is not visible anymore.
+        The iframe in which the test is currently operating is not visible.
```
Comment (inikulin) [inline src/errors/test-run/templates.js]: In which case we need this error?

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
Work on pull-request in progress. We've decided to change an approach. I'll write `fpr` when it will be ready

**Pull request closed without merging**